### PR TITLE
Connection signup page

### PR DESCRIPTION
**Pull request by Vaishnavi23032001**
API connection for login, userprofile, add event are done

### CODE REVIEW

File: src/Pages/UserProfile.jsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        const response = await axios.get(apiurl + "/api/users/" + userId, {
+          headers: {
+            'ngrok-skip-browser-warning': '111'
+          }
+         });
+        setUserProfile(response.data);
+      } catch (error) {
+        console.log(error);
+        toast.error("Error!", { position: "top-center" });
+      }
```
Comment (TjGanesh) [inline src/Pages/UserProfile.jsx:29]: API calls should be placed in seperate folder/file and the method should be called when needed

**Review by TjGanesh — COMMENTED** [review]
please make the changes mentioned

**Comment by Vaishnavi23032001** [comment]
Done the changes

File: src/Pages/UserProfile.jsx (hunk 1 above)
Comment (Vaishnavi23032001) [reply src/Pages/UserProfile.jsx:29]: Done

**Pull request merged**